src/test/java/PageObject/BasePage.java

```diff
@@ -0,0 +1,29 @@
+package PageObject;
```

Comment:
A package in Java is used to group related classes. 

For example package "pages" - Includes the Page Objects (or Page Classes)